commands/eureka-tasks.js

```diff
@@ -13,16 +13,20 @@ const printError = require('../errors/print-error')
 tasks.getTasks()
   .then(response => {
     const table = new Table({
-      head: ['Task Name', 'Machine', 'Status', 'Command', 'Tier', 'Start Time', 'Cost', 'Duration']
+      head: ['Machine', 'Status', 'Command', 'Tier', 'Cost', 'Duration'],
+      style: {
+        head: ['green'],
+        compact: true
+      }
     })
 
     for (const task of response) {
```

Comment:
I think we should sort order the table by initialization timestamp. You could just do a `Array.prototype.sort`